azimuth_caas_operator/utils/ansible_runner.py

```diff
@@ -98,10 +181,6 @@ def get_env_configmap(
         extraVars["cluster_state"] = "absent"
 
     envvars = dict(cluster_type_spec.envVars)
-    try:
-        envvars["CONSUL_HTTP_ADDR"] = os.environ["CONSUL_HTTP_ADDR"]
-    except KeyError:
-        raise RuntimeError("CONSUL_HTTP_ADDR is not set")
```

Comment:
I am tempted to keep this, but instead don't fail when the key is missing? Just to help the transition between the versions?